jsonapi-fetchdata.gemspec

```diff
@@ -31,6 +31,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'rspec', '~> 3.0'
 
-  spec.add_dependency 'activesupport', '~> 4.0'
+  spec.add_dependency 'activesupport', '~> 3.0'
```

Comment:
Since rails 4 is the largest install base, i dont want to require them to also install version 3.*
Altho I dont think the methods have changed overtime. So lets use '>= 3.0'